_config.yml

```diff
@@ -107,7 +107,7 @@ secondary_navigation:
 # 3. Add your site/affiliate name here.
 searchgov:
 
-  # You should not change this.
+  # only change this if you're using a CNAME, learn more here: https://search.gov/manual/cname.html
```

Comment:
@amanifarooque can you capitalize here please.

Reply:
Capitalized all the Search instructions!